src/components/vm/consoles/consoles.jsx

```diff
@@ -81,6 +80,38 @@ export class ConsoleCardState extends StateObject {
     }
 }
 
+export class ConsoleCardStates {
+    states = {};
+    accessTimes = {};
+    time = 0;
+
+    get(vm) {
+        const key = `${vm.connectionName}:${vm.name}`;
+
+        if (!this.states[key]) {
+            this.#makeRoom(Math.max(0, VMS_CONFIG.MaxConsoleCardStates - 1));
+            this.states[key] = new ConsoleCardState();
+        }
+
+        this.accessTimes[key] = this.time++;
+        return this.states[key];
+    }
+
+    #makeRoom(max) {
+        const keys = Object.keys(this.states);
+        if (keys.length > max) {
+            const sortedKeys = keys.sort((a, b) => (this.accessTimes[a] || 0) - (this.accessTimes[b] || 0));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2093-34c760d7-20250902-080839-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)